build_projects/dotnet-cli-build/project.json

```diff
@@ -20,7 +20,8 @@
     "System.Runtime.Serialization.Primitives": "4.1.1",
     "System.Xml.XmlSerializer": "4.0.11",
     "WindowsAzure.Storage": "6.2.2-preview",
-    "NuGet.CommandLine.XPlat": "3.5.0-rc1-1697",
+    "NuGet.Build.Tasks": "3.6.0-beta.1.msbuild.1",
```

Comment:
Our private build tasks project should need to reference `NuGet.Build.Tasks`.  Why is this needed?